.github/workflows/ci-tests.yml

```
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-20.04"]
-        python-version: ["3.9"]
+        python-version: ["3.10"]
```

```suggestion
        python-version: ["3.9"]
```
seems that we gave some failing tests early PT and newer python